ports/sleef/portfile.cmake

```
@@ -0,0 +1,29 @@
+# x86: Reports ICE(Internal Compiler Error) for x86-windows/release.
+#      see ${SOURCE_PATH}/src/libm/sleefsimddp.c
+# uwp: codegen fails
+vcpkg_fail_port_install(ON_ARCH "arm" "x86" ON_TARGET "uwp")
```

```suggestion
vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
```

Since this is a temporary error caused by a compiler bug, there's no reason to force it to be uninstallable

---

Also, this works on ARM, at least on osx...